### PR TITLE
Add translation about / resources in Spanish

### PR DESCRIPTION
**Pull request by Beor18**
Hello community, 

I would like to be able to make my first collaboration with a translation of about / resources in Spanish.

From already thank you very much!

### CODE REVIEW

File: locale/es/about/resources.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: Logos y Gráficos
+---
+# Recursos
+
+## Descargas de Logo
+
+ Por favor revise la [Política de Marcas Registradas](/about/trademark/) para obtener información sobre el uso permitido de Node.js&reg; logos y marcas.
+
+ Las pautas para la visualización de la marca Node.js se describen en
+ la [Pautas Visuales](/static/documents/foundation-visual-guidelines.pdf).
```
Comment (Tsuna-mi) [inline locale/es/about/resources.md]: Hi @Beor18, this article seems to be in plural

File: locale/es/about/resources.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+<table border="0" cellspacing="0" cellpadding="10" class="logos">
+  <tr>
+    <td bgcolor="#FFFFFF"><a href="/static/images/logos/nodejs-new-pantone-black.ai"><img src="/static/images/logos/nodejs-new-pantone-black.png" alt="Node.js on light background"></a></td>
```
Comment (Tsuna-mi) [inline locale/es/about/resources.md]: It would be nice to also translate images 'alt' atributes.

File: locale/es/about/resources.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <td bgcolor="#333333"><a href="/static/images/logos/nodejs-new-pantone-white.ai"><img src="/static/images/logos/nodejs-new-pantone-white.png" alt="Node.js on dark background"></a></td>
+  </tr>
+  <tr>
+    <td><a href="/static/images/logos/nodejs-new-pantone-black.ai">Node.js estandar AI</a></td>
```
Comment (Tsuna-mi) [inline locale/es/about/resources.md]: 'Estándar' needs accent. Review is more than one, please.

**Review by Tsuna-mi — CHANGES_REQUESTED** [review]
Thanks for your contribution @Beor18 
Review my littles comments 😌

File: locale/es/about/resources.md (hunk 2 above)
Comment (Beor18) [reply locale/es/about/resources.md]: thanks to you! :)

**Review by Tsuna-mi — APPROVED** [review]
Cool @Beor18 
Thanks

**Pull request merged**